pkg/security/fake/fake.go

```diff
@@ -1,3 +1,6 @@
+//go:build unit
```

Comment:
I added these so the files don't accidentally get built into the final binary. `unit` is the tag we use for all code that is only in unit tests